dandi/publish/tests/test_stats.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_stats_none(api_client):
+    resp = api_client.get('/api/stats/')
+    assert {
+        'draft_count': 0,
+        'published_count': 0,
+        'user_count': 0,
+        'species_count': 0,
+        'subject_count': 0,
+        'cell_count': 0,
+        'size': 0,
+    } == resp.json()
+
+
+@pytest.mark.django_db
+def test_stats_draft_count(api_client, draft_version_factory):
+    draft_version_factory()
```

Comment:
```suggestion
def test_stats_draft_count(api_client, draft_version):
```

pytest-factoryboy gives us a fixture which instantiates an actual model from the factory: https://pytest-factoryboy.readthedocs.io/en/latest/#model-fixture . You should only need to inject the factory explicitly if you want to pass different parameters to it.